lib/screens/application.ex

```diff
@@ -37,4 +38,34 @@ defmodule Screens.Application do
     ScreensWeb.Endpoint.config_change(changed, removed)
     :ok
   end
+
+  @doc """
+  Fetches a configuration value and raises if missing.
+
+  ## Examples
+
+      iex> Screens.Application.config(ScreensByAlert.Memcache, :connection_opts)
+      [
+        namespace: "localhost:4000",
+        hostname: "localhost"
+      ]
+  """
+  def config(root_key, sub_key) do
```

Comment:
Not a huuuuge fan of this custom convenience function, as it hides the fact that it calls `Application.fetch_env!` (which should only be called at runtime), while sometimes we want to call `Application.compile_env`, for example when we're storing some config in a module attribute.

[Application.compile_env](https://hexdocs.pm/elixir/Application.html#compile_env/3) also lets you provide a "path" to search for under a key, so this custom function shouldn't be necessary in that case.

Reply:
Ahh, I didn't think about that. Looked nifty so I gave it a shot. Ha. Changed it to just use the Elixir `Application` module.